.werft/installer-tests.ts

```diff
@@ -144,8 +174,9 @@ function getKubeconfig() {
 export async function installerTests(config: TestConfig) {
     console.log(config.DESCRIPTION);
     for (let phase of config.PHASES) {
-        const phaseSteps = INFRA_PHASES[phase];
-        const ret = callMakeTargets(phaseSteps.phase, phaseSteps.description, phaseSteps.makeTarget);
+        const args = phase.split(" ");
```

Comment:
I don't understand which phase has spaces in the name and needs to be split. Can you point me to the phase that's depending on this logic?

Reply:
This is a good question. This is my bad. Actually along with the upgrade I ended up adding some of the changes necessary for supporting different combinations of configurations in this one. So the thing is, depending on the `TEST_CONFIGURATION`, we can have different combinations of dependencies. like GKE with GCR or EKS with s3 etc. These things matter only when create a kots config file because that's where we specify all these connections. So I thought, I can pass args like "storage=s3" to the make target for `generate-kots-config`. But the problem is since these depend on the platform, I had to go back one more step and provide these args in `PHASES`. There are no phases in this example that does it, I just wanted to add the setup for it. Going one step further, for GKE the storage can be either `s3` or `in-cluster` and this can be chosen randomly so eventually we test all combinations. So I decided to do something like `storage=sample["s3, "in-cluster"]`. This randomly chooses one of the two storage options. Again, I know this is not the cleanest of solutions, but I wanted to have a working solution in place.